dist/immutable.js

```diff
@@ -3922,6 +3924,9 @@
   RecordPrototype.withMutations = MapPrototype.withMutations;
   RecordPrototype.asMutable = MapPrototype.asMutable;
   RecordPrototype.asImmutable = MapPrototype.asImmutable;
+  UNSUPPORTED_OPS.forEach(function(op)  {
+    RecordPrototype[op] = undefined;
```

Comment:
I wonder if it may make sense to fail loudly with these for now (i.e. explicitly throw an error stating the operation is unsupported) as opposed to silently removing them by simply setting them as undefined